### DISCUSSION TITLE
The Min browser is great! But...

### DISCUSSION THREAD

**Discussion by lpossamai** [minbrowser/min#1988] [General]
Yeah, it is great. It's fast, lightweight and the idea of not using the mouse to things, it really does feel like I'm in an `I3wm` browser.

However, it is far from being the only browser installed in my laptop. Don't take the suggestions/comments below personally, I'm just thinking out loud here, as I would LOVE to dump the other browsers (in my mobile phone too!!) and rely entirely on Min.

1. Support [multi-window](https://github.com/minbrowser/min/discussions/1982) is a must. Sometimes you just want to open a new window somewhere.
2. I miss extensions - I have to use some extensions for work, for example.
3. [Netflix](https://github.com/minbrowser/min/issues/641), Prime, DisneyPlus don't work
4. [Updated 1Password CLI to the latest version](https://github.com/minbrowser/min/issues/1981#issuecomment-1112849418) -It allows you to biometric unlock 1Password
5.  No mobile app
6. No sync (specially for bookmarks) between Min Browsers
7. No easy way to reorganize bookmarks

I really love the work done so far and have became a sponsor. Looking forward to the day I can go fully Min! :)

**Comment by flightmansam** [comment]
> 1. Support multi-window is a must. Sometimes you just want to open a new window somewhere.

I haven't worked any more on my current multi window branch. However I have used it a bunch this week!

> 2. I miss extensions - I have to use some extensions for work, for example.

Such a highly requested feature! But I am not free enough to have a look at this so I'll leave it for the head honch.

> 5. No mobile app

Electron doesn't support mobile so it would probably have to be a fork of min that is rewritten in a different framework (React Native, Native Script, Ionic). Also I believe some of the most powerful features of min are power-user keyboard based. Not sure how this would translate to a phone without getting bloated. Maybe gestures?

> 6. No sync (specially for bookmarks) between Min Browsers

There is some chat about this here: https://github.com/minbrowser/min/issues/1861

**Reply by lpossamai** [reply]
> Electron doesn't support mobile so it would probably have to be a fork of min that is rewritten in a different framework (React Native, Native Script, Ionic). Also I believe some of the most powerful features of min are power-user keyboard based. Not sure how this would translate to a phone without getting bloated. Maybe gestures?

yeah, I get that... but the idea of sharing bookmarks and/or sending a tab to your mobile phone and vice-versa is strong to me.. I use that all the time. So I reckon a mobile version is a must.

**Reply by PalmerAL** [reply]
On macOS and Windows, we could have a share button in the address bar, which would let you use airdrop or the Windows equivalent to send a link to your phone. Would that be helpful?

A true mobile version is unlikely to happen, because there isn't much from the desktop version that can be re-used, so we'd be writing and maintaining two entirely different browsers. I don't have the time available to do that; if someone else was interested in working on that, it might be possible, but it's a lot of work to do.

**Reply by flightmansam** [reply]
I have noticed that Firefox and Chrome can both make use of Safari (iOS) handoff requests. There is something in their plist that says they are compatible. I would love this functionality too. But it's very Mac specific. Let me know what you think and I'll do a bit more digging.

**Comment by flightmansam** [comment]
8. Some websites think we are bots and block access even if you attempt the captcha a few times. Maybe something to do with me using a custom user agent to get GServices working. E.g www.octopart.com

**Reply by PalmerAL** [reply]
It's probably because of the custom UA. What did you have to set it to to get Google working?

(We have a UA override for Google already: https://github.com/minbrowser/min/blob/master/main/UASwitcher.js#L49 . But if there's a better one than what we're using, we could switch).

**Reply by flightmansam** [reply]
I found that this worked for the GServices. But maybe it is illegal in other circumstances...?

`Mozilla/5.0 (Macintosh; Arm Mac OS X 10.15; rv:95.0) Gecko/20100101 Firefox/95.0`

**Comment by Pablo-fdez** [comment]
There are other browsers that do everything you ask for and more (e.g. Vivaldi). The appeal of min is that it prevents clutter by default by not allowing extensions and not allowing multiple windows.

**Reply by lpossamai** [reply]
Firefox, Chrome, Brave... they all do what I asked for. But that's not the point. The point is that I like how Min works, how lightweight it is and how it performs, and would like to make it my default browser and uninstall the other ones I mentioned.

And at the moment, I cannot do that.
